app/ts/ReportingUI/components/Reporting.tsx

```diff
@@ -395,40 +396,37 @@ export const Reporting = ({ maybeReadClient, maybeWriteClient, universe, reputat
-			const allPayoutNumerators = getAllPayoutNumeratorCombinations(hotLoadingMarketData.numOutcomes, hotLoadingMarketData.numTicks)
-			const winningOption = await getWinningPayoutNumerators(readClient, marketAddress.deepValue)
-			const winningIndex = winningOption === undefined ? -1 : allPayoutNumerators.findIndex((option) => areEqualArrays(option, winningOption))
-			const stakes = await getStakesOnAllOutcomesOnYesNoMarketOrCategorical(readClient, marketAddress.deepValue, hotLoadingMarketData.numOutcomes, hotLoadingMarketData.numTicks)
-			const alreadyContributedToOutcomes = await getAlreadyContributedCrowdSourcerInfoOnAllOutcomesOnYesNoMarketOrCategorical(readClient, marketAddress.deepValue, hotLoadingMarketData.numOutcomes, marketData.deepValue.hotLoadingMarketData.numTicks)
-			outcomeStakes.deepValue = stakes.map((repStake, index) => {
-				const payoutNumerators = allPayoutNumerators[index]
-				if (payoutNumerators === undefined) throw new Error(`outcome did not found for index: ${ index }. Outcomes: [${ hotLoadingMarketData.outcomes.join(',') }]`)
-				return {
-					outcomeName: getOutComeName(payoutNumerators, currentMarketData),
-					repStake,
-					status: index === winningIndex ? 'Winning' : 'Losing',
-					payoutNumerators,
-					alreadyContributedToOutcomeStake: alreadyContributedToOutcomes[index]?.stake
-				}
-			})
-		} else {
-			// Scalar market
-			const reportingParticipants = await getReportingParticipantsForMarket(readClient, marketAddress.deepValue)
-			const allKnownPayoutNumerators = aggregateByPayoutDistribution(reportingParticipants)
-			const winningOption = await getWinningPayoutNumerators(readClient, marketAddress.deepValue)
-			const winningIndex = winningOption === undefined ? -1 : allKnownPayoutNumerators.findIndex((option) => areEqualArrays(option.payoutNumerators, winningOption))
-			outcomeStakes.deepValue = allKnownPayoutNumerators.map((info, index) => {
-				const payoutNumerators = info.payoutNumerators
-				return {
-					outcomeName: getOutComeName(payoutNumerators, currentMarketData),
-					repStake: info.stake,
-					status: index === winningIndex ? 'Winning' : 'Losing',
-					payoutNumerators,
-					alreadyContributedToOutcomeStake: undefined, // TODO how to get?
+
+		const getAllInterestingPayoutNumerators = async() => {
+			const reportingParticipants = await getReportingParticipantsForMarket(readClient, currentMarketData.marketAddress)
+			switch (hotLoadingMarketData.marketType) {
+				case 'Categorical':
+				case 'Yes/No': {
+					// its possible for Augur to have "malformed payout numerators" being reported. Such as you can report 80% yes and 20% no on Yes/No market.
+					// We get these (along with valid ones that exist in the data) with `getReportingParticipantsForMarket`
+					// we merge all valid ones with all existing ones to get all interesting (as in either reported ones, or ones that make sense to report for) reporting options
```

Comment:
DO we still display any malformed reporting somewhere so people can see if the system is being attacked that way?

Reply:
yeah they are listed like normal options with name `Malformed Outcome (${ payoutNumerators.join(', ') })`